CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Onboarding/protocols/CustomResourceProvider.swift

```diff
@@ -0,0 +1,11 @@
+//
+//  CustomResourceProvider.swift
+//
+//  Copyright © 2024 Gini GmbH. All rights reserved.
+//
+
+import UIKit
+
```

Comment:
Could you please add a documentation here?

Reply:
Done :D.